aes-kw/src/lib.rs

```diff
@@ -431,15 +433,15 @@ impl<'a> BlockSizeUser for WCtx<'a> {
 
 /// Very similar to the W(S) function defined by NIST in SP 800-38F,
 /// Section 6.1
-impl<'a> BlockClosure for WCtx<'a> {
+impl<'a> BlockCipherEncClosure for WCtx<'a> {
     #[inline(always)]
-    fn call<B: BlockBackend<BlockSize = Self::BlockSize>>(self, backend: &mut B) {
+    fn call<B: BlockCipherEncBackend<BlockSize = Self::BlockSize>>(self, backend: &B) {
         for j in 0..=5 {
             for (i, chunk) in self.out.chunks_mut(SEMIBLOCK_SIZE).skip(1).enumerate() {
                 // A | R[i]
                 self.block[IV_LEN..].copy_from_slice(chunk);
                 // B = AES(K, ..)
-                backend.proc_block(self.block.into());
+                backend.encrypt_block(self.block.into());
```

Comment:
I *think* this is correct, but, dear reviewer, please pay attention to that bit.

Reply:
Looks correct to me